README.md

```diff
@@ -73,6 +73,18 @@ file has been signed by an authorized member of the io.js team.
 Once verified, use the SHASUMS256.txt.asc file to get the checksum for
 the binary verification command above.
 
+## Pre-built binaries vs source code
+
+This README.md file is included with both binaries and source tarballs.
+
+If you've downloaded a binary, the top-level folder will have a `bin`
+that contains pre-built executables for `iojs`, `node`, and `npm`,
+which you should be able to run from the command line.
```

Comment:
It seems like there needs to be another line about either:
- installing the bin, etc, include, lib, share into (say) /usr/local, or
- leaving things where they are but changing your environment (e.g., PATH) to find them there.